styles/layout.css

```diff
@@ -0,0 +1,179 @@
+body {
```

Comment:
Great job @chantal66! Maybe later we can refactor and separate some of this CSS into different files.